lib/src/models/user.dart

```diff
@@ -11,6 +11,7 @@ class User {
   final List<Profile> profiles;
   final String? referralLink;
   final bool tosAccepted;
+  final int? aiDailyMessageLimit;
```

Comment:
Let's make this an `int` instead of an `int?`. `_getAIDailyMessageLimitProperty` won't return `null`.